RELEASE.md

```diff
@@ -0,0 +1,79 @@
+This is an overview of how to run wgpu releases.
```

Comment:
style: backticks plz

suggestion: I'd like to distinguish `wgpu`-the-crate from `wgpu`-the-Git-repository.

```suggestion
This is an overview of how to run releases in the `wgpu` repository.
```